util-cgroups/src/measure/v2/mock.rs

```diff
@@ -323,4 +323,357 @@ nr_bursts 123
 burst_usec 123456789";
         assert_eq!(s, EXPECTED);
     }
+
+    #[test]
+    fn test_serialize_bool() {
+        let serializer = StatSerializer;
+        // Test with true
+        let true_serialized_res = serializer.serialize_bool(true);
+        assert!(true_serialized_res.is_ok());
+        let true_serialized = true_serialized_res.unwrap();
+        assert_eq!(true_serialized, "true");
+
+        let serializer = StatSerializer;
+        // Test with false
+        let false_serialized_res = serializer.serialize_bool(false);
+        assert!(false_serialized_res.is_ok());
+        let false_serialized = false_serialized_res.unwrap();
+        assert_eq!(false_serialized, "false");
```

Comment:
When you have a situation like this, it's more idiomatic to just call unwrap instead of assert is_ok:
```
assert_eq!(serializer.serialize_bool(false).unwrap(), "false");
```